webapp/service-worker-uninstaller.js

```diff
@@ -0,0 +1,8 @@
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.getRegistrations().then(registrations => {
```

Comment:
Since we are not accessing anything outside of this scope, function(registrations) { could be better?

Reply:
I'm not sure what you mean. In terms of scopes, arrow functions don't have `this`; both normal functions and arrow functions would capture lexical scopes and create closures.

I like it here because it's shorter ;)